packages/lesswrong/server/datadog/tracer.ts

```diff
@@ -0,0 +1,12 @@
+import tracer from "dd-trace";
+tracer.init({
+  hostname: process.env.IS_DOCKER ? "172.17.0.1" : undefined,
```

Comment:
172.17.0.1 is the gateway IP of the docker bridge network, and it is just [recommended](https://docs.datadoghq.com/integrations/amazon_elasticbeanstalk/?tab=singlecontainer) that you hardcode this. It's also useful to be able to run the datadog agent and collect traces locally for debugging, which is why I've added the IS_DOCKER check here